Automattic/wp-calypso
# Add/announcement modal

#### Changes proposed in this Pull Request

* Created a new Announcement modal block that can be reused in future cases
* Added the modal in the plugin browser landing page using a feature flag `marketplace`

#### Testing instructions



* Go to `/plugins`
* You should see the modal
* Click X or 'Let's explore' 
* Refresh the page, you shouldn't see the modal
* If you delete it from your preferences you may see it again 
![SS 2021-11-04 at 14 44 09](https://user-images.githubusercontent.com/12430020/140315572-afa9e4b4-62d1-4c64-b8d3-c6e4f95f459b.png)


<img width="1076" alt="SS 2021-11-04 at 14 44 42" src="https://user-images.githubusercontent.com/12430020/140315605-49425b2d-2ecb-43e9-a2f2-486e936923de.png">




Fixes #56955
